renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      "automerge": true,
```

Comment:
Can we also label PRs that will be automerged?  -- https://docs.renovatebot.com/configuration-options/#labels

Reply:
I think we don't want to use `automerge` because then kodiak will think it has to do something, and kodiak is installed at the org level. Is there some other label that would make sense to you? How about `dependencies`?

Reply:
ah good point. `dependencies` is fine with me